client/src/analytics/events.ts

```diff
@@ -79,4 +79,10 @@ export enum EVENTS {
   DOCUMENTATION_CLICK_NAV = "DOCUMENTATION_CLICK_NAV",
   CELL_GUIDE_CLICK_NAV = "CELL_GUIDE_CLICK_NAV",
   DE_CLICK_NAV = "DE_CLICK_NAV",
+
+  NEWSLETTER_SIGNUP_SUCCESS = "NEWSLETTER_SIGNUP_SUCCESS",
```

Comment:
I see that you have a reminder to add events to Plausible already 👍 Thanks so much!